tests/conftest.py

```diff
@@ -1,28 +1,110 @@
-import fnmatch
-import http.client
+import asyncio
 import json
 import logging
 import os
 import random
 import shutil
-import subprocess
-from time import sleep
+import socket
+from copy import deepcopy
+from pathlib import Path
+from subprocess import STDOUT, Popen, run
 
 import pytest
-from hopr import HoprdAPI
 
-random_data = os.urandom(8)
-SEED = int.from_bytes(random_data, byteorder="big")
+from .node import Node
+
+SEED = int.from_bytes(os.urandom(8), byteorder="big")
 random.seed(SEED)
 
+# Global variables
 LOCALHOST = "127.0.0.1"
-OPEN_CHANNEL_FUNDING_VALUE = "1000000000000000000000"
+OPEN_CHANNEL_FUNDING_VALUE = 1000
```

Comment:
That's a huge jump, why?

Reply:
this value is in HOPR, and later converted into weiHOPR in the api helper.
Helper exist to help, here to help us read this amount of token